### ISSUE TITLE
ERROR near_client::client] Error: Other("EpochOutOfBounds")

### ISSUE THREAD

**Issue by hugski** [nearprotocol/nearcore#1068]
when i run  
`./scripts/start_testnet.py --local`, then it was interrupt by network failure.

then i run this scripts agin, I encountered this issue, detailed outputs are below:

```
****************************************************
* Running NEAR validator node for Official TestNet *
****************************************************
cargo 1.37.0-nightly (4c1fa54d1 2019-06-24)
info: using existing install for 'nightly-x86_64-apple-darwin'
info: default toolchain set to 'nightly-x86_64-apple-darwin'

  nightly-x86_64-apple-darwin unchanged - rustc 1.37.0-nightly (8ebd67e4e 2019-06-27)

    Finished release [optimized] target(s) in 1.29s
Using existing node configuration: /Users/zpf/.near/
Stake for user 'tusking' with 'Ejuy1JoEujgFVq6iu2mcjAHTasunVnvAu5MmeavhDCKY'
Starting NEAR client...
Autoupdate is not supported at the moment for local run
[2019-07-12T09:11:42Z INFO  near] Opening store database at "/Users/zpf/.near/data"
[2019-07-12T09:11:42Z INFO  near_network::peer_manager] Server listening at 8H5eBkdYW9MMHoLyfdgqK3aNwaShQiPEio6a11mc3FRc@0.0.0.0:24567
[2019-07-12T09:11:52Z ERROR near_client::client] Error: Other("EpochOutOfBounds")
```

**Comment by pkrasam** [comment]
I am getting similar error today...

```
03:54:07 ✔ pk: (master) nearcore $ docker logs --follow nearcore
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/my_init.d/10_syslog-ng.init...
Jul 16 17:54:06 linuxkit-025000000001 syslog-ng[13]: syslog-ng starting up; version='3.13.2'
*** Booting runit daemon...
*** Runit started as PID 21
*** Running /usr/local/bin/run.sh...
Bootnodes: 49ppQ9vkLYvWajZ1KRMdism4AQswFT4yD2e9kRt7B4rC@34.94.33.164:24567
Jul 16 17:54:07 linuxkit-025000000001 cron[25]: (CRON) INFO (pidfile fd = 3)
Jul 16 17:54:07 linuxkit-025000000001 cron[25]: (CRON) INFO (Running @reboot jobs)
[2019-07-16T17:54:07Z INFO  near] Opening store database at "/srv/near/data"
[2019-07-16T17:54:07Z INFO  near_network::peer_manager] Server listening at FVa4pwU2kGUwauSGhCzuUmM4U6NyikQLseR8ovvBg1gf@0.0.0.0:24567
[2019-07-16T17:54:17Z ERROR near_client::client] Error: Other("EpochOutOfBounds")
```

**Comment by ilblackdragon** [comment]
This should have been addressed with 0.4 release.
